src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -432,7 +432,12 @@ uct_rdmacm_cm_ep_send_priv_data(uct_rdmacm_cm_ep_t *cep, const void *priv_data,
 err:
     uct_rdmacm_cm_ep_destroy_dummy_qp(cep);
     remote_data.field_mask = 0;
-    uct_rdmacm_cm_ep_set_failed(cep, &remote_data, status);
+
+    /* */
+    if ((cep->flags & UCT_RDMACM_CM_EP_ON_CLIENT) &&
+        (cep->super.priv_pack_cb != NULL)) {
+        uct_rdmacm_cm_ep_set_failed(cep, &remote_data, status);
+    }
```

Comment:
move this section to 412-415, assuming that we can do uct_rdmacm_cm_ep_destroy_dummy_qp(cep) after uct_rdmacm_cm_ep_set_failed(cep)

Reply:
done